### ISSUE TITLE
PV Bug Report - module is found unter permission viewer, not Ownership viever

### ISSUE THREAD

**Issue by vt-tom** [Malekal4699/fvtt-module-permission-viewer#45]
**Describe the bug**
For v10 at first the Permission Viewer was changed to Ownership Viewer. But now i can't find it under Ownership viewer, but Permission Viewer. 
Maybe some Changes in 10.285 caused this?

**To Reproduce**
1. Go to the module Update Page
2. Update Ownership Viewer (manifest error)
3. uninstall OV
4. try to search for OV
5. try to search for PV

**Expected behavior**
Find OV under the correct name.

**Screenshots**
![grafik](https://user-images.githubusercontent.com/80099175/190063374-b145e2c3-9ca7-49f6-a90e-449ebec0f2c8.png)

**Foundry Information & Browser (please complete the following information):**
 - Foundry version 10.285
 - Permission Viewer Version 0.10.0
 - Browser Crhome

**Comment by Doresain** [comment]
Yeah I changed the name as per the shift in Foundry Core in the PR for v10, but it needs to be updated on the package admin page.
